Library/Homebrew/test/support/fixtures/testball_bottle_cellar.rb

```diff
@@ -0,0 +1,24 @@
+# typed: true
+# frozen_string_literal: true
+
+class TestballBottleCellar < Formula
+  def initialize(name = "testball_bottle_cellar", path = Pathname.new(__FILE__).expand_path, spec = :stable,
+                 alias_path: nil, force_bottle: false)
+    self.class.instance_eval do
+      stable.url "file://#{TEST_FIXTURE_DIR}/tarballs/testball-0.1.tbz"
+      stable.sha256 TESTBALL_SHA256
+      hexdigest = "8f9aecd233463da6a4ea55f5f88fc5841718c013f3e2a7941350d6130f1dc149"
+      stable.bottle do
+        root_url "file://#{TEST_FIXTURE_DIR}/bottles"
+        sha256 hexdigest => Utils::Bottles.tag, cellar => :any_skip_relocation
```

Comment:
Invalid syntax; should be

```suggestion
        sha256 hexdigest => Utils::Bottles.tag, cellar: :any_skip_relocation
```

or

```suggestion
        sha256 hexdigest => Utils::Bottles.tag, :cellar => :any_skip_relocation
```

I'm not sure which syntax we've decided on

Reply:
Done. I would go for `:cellar => :any_skip_relocation`, that's what I have chosen.